spec/features/reply_from_councillor_spec.rb

```diff
@@ -13,9 +13,8 @@
     end
   end
 
-  background { comment.confirm! }
```

Comment:
More info re: removing this background here - for the WriteIt test there are 2 http requests potentially here:
1) When creating a comment 
GET http://maps.google.com/maps/api/geocode/json?address=24%20Bruce%20Road%20Glenbrook&region=au&sensor=false
2) When confirming a comment that is for WriteIt
POST http://writeit.ciudadanointeligente.org/api/v1/message/

These are currently in the same VCR cassette (planningalerts) and so would need to record a new cassette. Seemed simpler this way for now (and less changes).

As the confirm wasn't actually being tested for that test I avoided